### PR TITLE
Mrc 247 upgrade performance tests

### PR DESCRIPTION
**Pull request by conniebernardin**

### CODE REVIEW

File: src/test/resources/journeys.conf [hunk 1]
```diff
@@ -42,6 +42,10 @@ journeys {
       post-associated-companies-page,
       get-check-your-answers-page,
       get-results-page
+      get-full-results-page
```
Comment (girishkamat) [inline src/test/resources/journeys.conf]: are we missing the commas or is it optional?

Comment (conniebernardin) [reply src/test/resources/journeys.conf]: It seems to work with or without, but I could add them for consistency if you'd prefer?

File: src/test/scala/uk/gov/hmrc/perftests/example/MarginalReliefCalculatorRequestsRequests.scala [hunk 2]
```diff
@@ -118,5 +118,31 @@ object MarginalReliefCalculatorRequestsRequests extends ServicesConfiguration {
...
+  val postPDFMetaDataPage: HttpRequestBuilder =
+    http("Post pdf meta data page")
+      .post(s"$baseUrl$route/pdf-meta-data": String)
+      .formParam("Company name (optional)", "abcdefg")
+      .formParam("UTR number (optional)", "abcdefg")
+      .formParam("csrfToken", s"$${csrfToken}")
+      .check(status.is(303))
+      .check(header("Location").is(s"$route/pdf").saveAs("pdf"))
+
+  val PDFPage: HttpRequestBuilder =
```
Comment (girishkamat) [inline src/test/scala/uk/gov/hmrc/perftests/example/MarginalReliefCalculatorRequestsRequests.scala]: can we use the correct casing for the variable and also rename to viewPDF?

File: src/test/scala/uk/gov/hmrc/perftests/example/MarginalReliefCalculatorRequestsRequests.scala [hunk 3]
```diff
@@ -118,5 +118,31 @@ object MarginalReliefCalculatorRequestsRequests extends ServicesConfiguration {
...
+    http("Get PDF page")
+      .get(s"$baseUrl$${pdf}": String)
+      .check(status.is(200))
+
+
```
Comment (girishkamat) [inline src/test/scala/uk/gov/hmrc/perftests/example/MarginalReliefCalculatorRequestsRequests.scala:150]: Where is the download PDF call?

**Review by adamjw3 — APPROVED** [review]

**Pull request merged**